scripts/common/lib.sh

```diff
@@ -124,3 +124,18 @@ skip_if_companion_pr() {
 latest_release() {
   curl -s "$api_base/$1/releases/latest" | jq -r '.tag_name'
 }
+
+# Check for runtime changes between two commits. This is defined as any changes
+# to /primitives/src/* and any *production* chains under /runtime
+has_runtime_changes() {
```

Comment:
we also have https://github.com/paritytech/polkadot/blob/1008a7f3240d38e1bd3c3bbffe45fd49162e1ff7/scripts/gitlab/check_runtime.sh#L55, which is slightly different

worth merging the two?